### ISSUE TITLE
Remove Admin "settings" UX, all settings will be in ENV vars

### ISSUE THREAD

**Issue by chriskacerguis** [projectjellyfish/api#134]
We need to have admin settings modularized.  I'm thinking in this directory path.

config/mods/:mod_name/admin_settings/:setting_name.json

There could be 1 or more admin settings.

Also, we need to document that schema for the JSON file.

**Comment by mafernando** [comment]
put all the env variables in .env.sample for implementation reference and update readme

**Comment by mafernando** [comment]
Looks like Jeremy did some work on this at https://github.com/projectjellyfish/api/pull/237, so admin settings are now blank from the UX.

![image](https://cloud.githubusercontent.com/assets/9356425/7074826/04df6574-dec3-11e4-80bc-014b8dd8a878.png)

Per above direction, am investigating if we can use the default autoloading behavior of Rails and its engines to accomplish this task.

It is straightforward if we load the .env vars from the `config/initializers` dir, but am not sure if that works as cleanly for `/config/mods`.

**Comment by mafernando** [comment]
<div><center><table><tr><td><i>We need to have admin settings modularized. I'm thinking in this directory path.<br>
<br>
config/mods/:mod_name/admin_settings/:setting_name.json<br>
<br>
There could be 1 or more admin settings.<br>
<br>
Also, we need to document that schema for the JSON file.</i></td></tr></table></div>

**Comment by chriskacerguis** [comment]
@mafernando I'm thinking that this is no longer valid, as we are switching to just using ENV vars (RE: #134).  Thoughts?

**Comment by mafernando** [comment]
agreed @chriskacerguis - https://github.com/projectjellyfish/api/issues/359 and https://github.com/projectjellyfish/api/issues/134 are very similar. 

should we remove 'settings' as a navigable option from the admin side bar before closing this?

**Comment by chriskacerguis** [comment]
Yeah, let's do that.

**Comment by mafernando** [comment]
To Do:
- [ ] remove settings from admin side bar options

**Comment by mafernando** [comment]
See PR https://github.com/projectjellyfish/api/pull/499 for implementation.

**Comment by chriskacerguis** [comment]
We need to still do the following

- [ ] Document the ENV vars in the .env.sample
- [ ] Remove the database table 
- [ ] Remove the seed data

**Comment by mafernando** [comment]
So I deleted the `settings` table, but there are references to the table that need to be transitioned in API. 

For [example](https://github.com/projectjellyfish/api/blob/master/app/controllers/saml_controller.rb#L33), attempts to look up the SAML from the `settings` table on a basic page load.

Also need to remove the specs tied to the services controller.

**Comment by chriskacerguis** [comment]
@mafernando Where are the env vars documented?

**Comment by mafernando** [comment]
@chriskacerguis - [here](https://github.com/projectjellyfish/api/wiki/Jellyfish-Environment-Variables) is the documentation

**Comment by chriskacerguis** [comment]
@mafernando so in the database table that had all the settings there were a lot more than that, can you see if you can dig up the old settings table and get the settings from there too?

**Comment by mafernando** [comment]
I did a global find for all references to `Setting` and `setting` in the code and replaced those with references to the dotenv file. 

The old settings table was populated from [this](https://github.com/projectjellyfish/api/blob/master/db/seeds/settings.yml) seed file which I left in for reference. 

The dotenv [documentation](https://github.com/projectjellyfish/api/wiki/Jellyfish-Environment-Variables) was setup to only include references to settings that were actively being used in api/ux at the time of writing - I intentionally left out the settings that weren't being used to avoid cluttering the dotenv file. For example [header and footer](https://github.com/projectjellyfish/api/commit/513e8e51b54fe5d7b3c45118dcf28962a6b67ee8) were removed because I could not locate anywhere in the code where those settings and their subattributes (e.g. `FOOTER_LINK_1_LABEL` or `HEADER_LINK_1_LABEL`) were referenced. Moreover, all the specs passed after I decoupled references in api/ux to the old settings table - so I'm not sure which settings are left out. 

@chriskacerguis - are there any specific environment variables that have been left out or will documenting what's currently in the wiki in the `.env.sample` file be sufficient?

ToDo:
- [ ] update `.env.sample` with [documentation](https://github.com/projectjellyfish/api/wiki/Jellyfish-Environment-Variables) for default dotenv vars and values.
